alpine-base/Makefile

```diff
@@ -1,5 +1,5 @@
-include ../../Makefile.conf
 include ../Makefile.conf
+include ./Makefile.conf
```

Comment:
Same thing, just remove this file and set IMAGE=alpine-base in Makefile directly.